### DISCUSSION TITLE
how to disable html escaping in cue cli ?

### DISCUSSION THREAD

**Discussion by sahroshan** [cue-lang/cue#2225] [Q&A] [answered]
Hi,
I am trying to generate json configuration from cue. But it is html escaping by default. How can we disable html escaping behaviour in cue cli.

example:

hello.cue
`a: "<hello>"`

`cue export hello.cue --out json` generates 
```
{
    "a": "\u003chello\u003e"
}
```

**Comment by mvdan** [comment, marked as answer]
See https://github.com/cue-lang/cue/issues/1243, which is an open bug.

**Reply by sahroshan** [reply]
Thank you for response.
Is there no known workaround this?

**Reply by mvdan** [reply]
Not at the moment. The bug is still open.
